### DISCUSSION TITLE
TeraHz Channel Model

### DISCUSSION THREAD

**Discussion by cacent96** [NVlabs/sionna#63] [General]
Hello,

From the white paper "Sionna: An Open-Source Library for Next-Generation Physical Layer Research", you mentioned the TeraHz channel model is under development. Anywhere I could see the development progress? And approximate time to complete? 

Thank you!

**Comment by SebastianCa** [comment]
Please understand that we cannot provide a timeline for the THz channel mode at the moment. 
However, dedicated channel models for THz are definitively on our roadmap for future releases of Sionna.
